### PR TITLE
Reduce title length to make it readable when plugin is initially installed.

### PR DESCRIPTION
**Pull request by ivanporty**
When the plugin is initially installed into a fresh IDE never configured for usage statistics before, the notification is shown at the start, clipping the title at 3-4 words making it essentially unreadable.

### CODE REVIEW

File: google-cloud-core/src/com/google/cloud/tools/intellij/analytics/UsageTrackerNotification.java [hunk 1]
```diff
@@ -46,29 +46,26 @@ public static UsageTrackerNotification getInstance() {
...
-                  BrowserUtil.browse(new URL(UsageTrackerPanel.PRIVACY_POLICY_URL));
-                } catch (MalformedURLException ex) {
-                  LOG.error(ex);
-                }
-                notification.expire();
-              } else if ("settings".equals(description)) {
-                final ShowSettingsUtil util = ShowSettingsUtil.getInstance();
-                util.showSettingsDialog(null, UsageTrackerConfigurable.class);
-                notification.expire();
+        (notification, event) -> {
```
Comment (etanshaul) [inline google-cloud-core/src/com/google/cloud/tools/intellij/analytics/UsageTrackerNotification.java]: I think this is the case, but just to make sure I didn't miss anything -> this is just a formatting change (and lambda refactoring) right?

**Comment by etanshaul** [comment]
could you post a screenshot possibly?

File: google-cloud-core/src/com/google/cloud/tools/intellij/analytics/UsageTrackerNotification.java (hunk 1 above)
Comment (ivanporty) [reply google-cloud-core/src/com/google/cloud/tools/intellij/analytics/UsageTrackerNotification.java]: right

**Comment by ivanporty** [comment]
Need to re-create new user and install fresh IDE - only guaranteed option to see it again, then I'll try to catch a screenshot.

**Comment by ivanporty** [comment]
That's how it looked originally, so shorter title and line-by-line text should make it more readable.

Screen Shot 2019-03-11 at 5 36 34 PM

**Pull request merged**